include/alpaka/queue/Traits.hpp

```diff
@@ -40,6 +40,14 @@ namespace alpaka
                 typename TQueue,
                 typename TSfinae = void>
             struct Empty;
+
+            //#############################################################################
+            //! Queue for an accelerator
+            template<
+                typename TAcc,
+                typename TProperty,
+                typename TSfinae = void>
+            struct Queue;
```

Comment:
To be consistent with all the other traits this would have to be called `QueueType`. Only the public alias templates are called `Queue` (without `Type`).